recipesesame/src/main/java/com/recipesesame/Main.java

```diff
@@ -11,17 +11,21 @@
 
 public class Main {
     public static void main( String[] args ) throws IOException, ClassNotFoundException, RecipeNotFoundException {
-        System.out.println(System.getProperty("user.dir"));
-        Database database = new FileSystemDatabase("RecipeSesame/recipesesame/src/main/java/com/recipesesame/recipes");
-        ArrayList<Recipe> recipes = database.getAllRecipes();
-    	
+        // System.out.println(System.getProperty("user.dir"));
         Scanner scan = new Scanner(System.in);
         BufferedOutputStream out = new BufferedOutputStream(System.out);
+
+        System.out.print("Your recipe folder path: ");
```

Comment:
In this case, the user would have to already know what the working directory is and type out the whole filepath every time.

Reply:
I was thinking we can show on the confluence how to copy the recipe path by right clicking on the folder through the ide as a work around to changing the code

Reply:
Makes sense